### PR TITLE
docs(rasterize): added openapi doc

### PR DESCRIPTION
**Pull request by melancholiai**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: openapi3.yaml [hunk 1]
```diff
@@ -1,64 +1,101 @@
 openapi: 3.0.1
 info:
-  title: service-name
-  description: basic template for map colonies service
+  title: vector-tiles-rasterizer
+  description: A proxy server for rasterizing vector tiles
   version: 1.0.0
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
```
Comment (syncush) [inline openapi3.yaml]: it should match mapbox gl native license

File: openapi3.yaml [hunk 2]
```diff
@@ -1,64 +1,101 @@
...
+    tilePosition:
+      type: number
+      format: int32
+      default: 0
+      minimum: 0
     error:
       type: object
       required:
         - message
+        - error
```
Comment (syncush) [inline openapi3.yaml]: what is the difference between `error` and message `fields`?

**Review by syncush — CHANGES_REQUESTED** [review]

File: openapi3.yaml (hunk 1 above)
Comment (melancholiai) [reply openapi3.yaml]: done.

File: openapi3.yaml (hunk 2 above)
Comment (melancholiai) [reply openapi3.yaml]: [see this](https://www.fastify.io/docs/latest/Reply/#errors), for some reason the field `code` is not being returned. not that we want it.
so there are three fields. `error` is the http status code in words while `message` is a custom message
```json
{
  "statusCode": 500,
  "error": "Internal Server Error",
  "message": "blabla"
}`

**Review by syncush — APPROVED** [review]

**Pull request merged**